packages/manager/apps/web/client/app/hosting/email/detach/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "hosting_email_detach_option_title": "Délier mon option e-mail",
+  "hosting_email_detach_option_description": "Votre option e-mail est actuellement liée à l'hébergement web {{hosting}}. Délier l‘option vous permet de la renouveler de manière indépendante de votre hébergement. L‘option vous sera désormais facturée au prix standard des options e-mail.",
+  "hosting_email_detach_option_error": "Une erreur est survenue lors du détachement de l‘option. {{error}}",
+  "hosting_email_detach_option_success_with_payment": "Le détachement de l‘option e-mail a bien été pris en compte. Votre compte {{accountId}} a bien été débité de {{price}}. <a class=\"d-block\" target=\"_blank\" href=\"{{billUrl}}\" rel=\"noopener\">Voir ma facture</a>",
```

Comment:
Maybe the link could be extracted ? Looks like its in its own sentence

Reply:
Yes it is its own sentence, but the link has to be open from `Alerter.success` so…